recipes/nvcloth/1.1.6/conanfile.py

```diff
@@ -0,0 +1,194 @@
+            tools.patch(**patch)
+        
+        if self.settings.build_type == "Debug":
+            shutil.copy(
+                os.path.join(self.build_folder, self._source_subfolder, "NvCloth/include/NvCloth/Callbacks.h"),
+                os.path.join(self.build_folder, self._source_subfolder, "NvCloth/include/NvCloth/Callbacks.h.patched")
+            )
+            shutil.copy(
+                os.path.join(self.build_folder, self._source_subfolder, "NvCloth/include/NvCloth/Callbacks.h.origin"),
+                os.path.join(self.build_folder, self._source_subfolder, "NvCloth/include/NvCloth/Callbacks.h")
+            )
+
+    def build(self):
+        os.environ['GW_DEPS_ROOT'] = os.path.abspath(self._source_subfolder)
+        self._copy_sources()
+        self._patch_sources()
+        self._remove_samples()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def _get_build_type(self):
+        if self.settings.build_type == "Debug":
+            return "debug"
+        elif self.settings.build_type == "RelWithDebInfo":
+            return "checked"
+        elif self.settings.build_type == "Release":
+            return "release"
+    
+    def _get_target_build_platform(self):
+        return {
+            "Windows" : "windows",
+            "Linux" : "linux",
+            "Macos" : "mac",
+            "Android" : "android",
+            "iOS" : "ios"
+        }.get(str(self.settings.os))
+
+    def package(self):
+        if self.settings.build_type == "Debug":
+            shutil.copy(
+                os.path.join(self.build_folder, self._source_subfolder, "NvCloth/include/NvCloth/Callbacks.h.patched"),
+                os.path.join(self.build_folder, self._source_subfolder, "NvCloth/include/NvCloth/Callbacks.h")
+            )
+        nvcloth_source_subfolder = os.path.join(self.build_folder, self._source_subfolder)
+        nvcloth_build_subfolder = os.path.join(self.build_folder, self._build_subfolder)
+
+        self.copy(pattern="NvCloth/license.txt", dst="licenses", src=nvcloth_source_subfolder, keep_path=False)
+        self.copy("*.h", dst="include", src=os.path.join(nvcloth_source_subfolder, "NvCloth", "include"))
+        self.copy("*.h", dst="include", src=os.path.join(nvcloth_source_subfolder, "NvCloth", "extensions", "include"))
+        self.copy("*.h", dst="include", src=os.path.join(nvcloth_source_subfolder, "PxShared", "include"))
+        self.copy("*.a", dst="lib", src=nvcloth_build_subfolder, keep_path=False)
+        self.copy("*.lib", dst="lib", src=nvcloth_build_subfolder, keep_path=False)
+        self.copy("*.dylib*", dst="lib", src=nvcloth_build_subfolder, keep_path=False)
+        self.copy("*.dll", dst="bin", src=nvcloth_build_subfolder, keep_path=False)
+        self.copy("*.so", dst="lib", src=nvcloth_build_subfolder, keep_path=False)
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
```suggestion
        self.cpp_info.libs = tools.collect_libs(self)
```

Avoid collect_libs, prefer declaring the expected libraries names, so anyone can understand it better.